x-pack/plugin/ilm/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/indexlifecycle/CCRIndexLifecycleIT.java

```diff
@@ -374,6 +367,20 @@ public void testCannotShrinkLeaderIndex() throws Exception {
                     assertILMPolicy(client(), indexName, policyName, "hot", "unfollow", "wait-for-indexing-complete");
                 });
 
+                // Index a bunch of documents and wait for them to be replicated
+                for (int i = 0; i < 50; i++) {
+                    index(leaderClient, indexName, Integer.toString(i));
+                }
+                assertBusy(() -> {
+                    for (int i = 0; i < 50; i++) {
+                        index(client(), indexName, Integer.toString(i));
```

Comment:
I'm not sure I follow this, we used to index a document on the leader and then make sure it existed on the follower, but now we index 50 documents on the leader and then index 50 documents on the follower, do we need to change this to check for the documents' existence?

Reply:
You are absolutely correct. Fixed.